packages/webdriver/protocol/chromium.json

```diff
@@ -1,169 +1,356 @@
 {
+  "/session/:sessionId/alert": {
+    "GET": {
+      "command": "isAlertOpen",
+      "description": "Whether a simple dialog is currently open.",
+      "ref": "https://github.com/bayandin/chromedriver/blob/2.45/alert_commands.cc#L42-L49",
+      "examples": [
+        [
+          "console.log(await browser.isAlertOpen()); // outputs: false",
+          "await browser.execute('window.alert()');",
+          "console.log(await browser.isAlertOpen()); // outputs: true"
```

Comment:
For consistency I would propose to remove all `awaits` in the example. The [API section](https://webdriver.io/docs/api.html#examples) explains that all examples are written with wdio-sync enabled.